### ISSUE TITLE
Startup crash (0x1b753cb38)

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero-ios#589]
https://forums.zotero.org/discussion/101371/zotero-crashed-on-ipad-and-deleted-some-pdfs

```
Exception Type:  SIGABRT
Exception Codes: #0 at 0x1b753cb38
Crashed Thread:  0

Thread 0 Crashed:
0   libsystem_kernel.dylib              0x00000001b753cb38 0x1b7535000 + 31544
1   libsystem_c.dylib                   0x000000018b7e0524 0x18b7c0000 + 132388
2   Zotero                              0x0000000104cb32c4 0x10420c000 + 11170500
3   Zotero                              0x0000000104cb3604 0x10420c000 + 11171332
4   Zotero                              0x0000000104cb3380 0x10420c000 + 11170688
5   Zotero                              0x0000000104a4f384 0x10420c000 + 8663940
6   Zotero                              0x0000000104a4f294 0x10420c000 + 8663700
7   Zotero                              0x0000000104ac784c 0x10420c000 + 9156684
8   Zotero                              0x000000010485a2f8 0x10420c000 + 6611704
9   Zotero                              0x00000001049c6b74 0x10420c000 + 8104820
10  Zotero                              0x00000001049fce84 0x10420c000 + 8326788
11  Zotero                              0x00000001049fd9cc 0x10420c000 + 8329676
12  Zotero                              0x0000000104253878 0x10420c000 + 292984
13  Zotero                              0x000000010424fb18 0x10420c000 + 277272
14  Zotero                              0x00000001045b9f24 0x10420c000 + 3858212
15  Zotero                              0x00000001045bb4a0 0x10420c000 + 3863712
16  UIKitCore                           0x0000000182d399c4 0x182ab3000 + 2648516
17  UIKitCore                           0x0000000182f37030 0x182ab3000 + 4735024
18  UIKitCore                           0x0000000182d5ebbc 0x182ab3000 + 2800572
19  UIKitCore                           0x0000000182c2eed8 0x182ab3000 + 1556184
20  UIKitCore                           0x0000000182c3f17c 0x182ab3000 + 1622396
21  QuartzCore                          0x00000001843b77fc 0x184377000 + 264188
22  QuartzCore                          0x00000001843a9c60 0x184377000 + 207968
23  QuartzCore                          0x00000001843be5b4 0x184377000 + 292276
24  QuartzCore                          0x00000001843c74a8 0x184377000 + 328872
25  QuartzCore                          0x00000001843a93a0 0x184377000 + 205728
26  UIKitCore                           0x0000000182ff16e0 0x182ab3000 + 5498592
27  UIKitCore                           0x000000018328bd5c 0x182ab3000 + 8228188
28  UIKitCore                           0x0000000183912edc 0x182ab3000 + 15072988
29  UIKitCore                           0x00000001839126a4 0x182ab3000 + 15070884
30  CoreFoundation                      0x00000001806f0414 0x180635000 + 766996
31  CoreFoundation                      0x00000001807011a0 0x180635000 + 836000
32  CoreFoundation                      0x000000018063a694 0x180635000 + 22164
33  CoreFoundation                      0x000000018064005c 0x180635000 + 45148
34  CoreFoundation                      0x0000000180653bc8 0x180635000 + 125896
35  GraphicsServices                    0x000000019c7bf374 0x19c7be000 + 4980
36  UIKitCore                           0x0000000182fc7b58 0x182ab3000 + 5327704
37  UIKitCore                           0x0000000182d49090 0x182ab3000 + 2711696
38  Zotero                              0x0000000104211c3c 0x10420c000 + 23612
39  ???                                 0x0000000105345da4 0x0 + 0
```

**Comment by michalrentka** [comment]
https://github.com/realm/realm-swift/issues/8045

**Comment by michalrentka** [comment]
Should be fixed in 66ac152a28fb5d5f294e5fdc2bd65605a258b365, affected users have to re-install unfortunately, but we shouldn't see new cases of this.
